Tribler/Test/test_libtorrent_download.py

```diff
@@ -9,12 +9,12 @@
 from Tribler.Core.simpledefs import DOWNLOAD
 
 
-TORRENT_R = r'http://torrent.fedoraproject.org/torrents/Fedora-Live-Workstation-x86_64-21.torrent'
-TORRENT_INFOHASH = binascii.unhexlify('89f0835dc2def218ec4bac73da6be6b8c20534ea')
+TORRENT_R = r'http://torrent.ubuntu.com/releases/wily/release/server/ubuntu-15.10-server-ppc64el.iso.torrent'
+TORRENT_INFOHASH = binascii.unhexlify('9874df3783f8de52f944032b9affe4a5c88bc9fe')
 TORRENT_FILE = os.path.join(TESTS_DATA_DIR, "ubuntu-15.04-desktop-amd64.iso.torrent")
 TORRENT_FILE_INFOHASH = binascii.unhexlify("fc8a15a2faf2734dbb1dc5f7afdc5c9beaeb1f59")
 TORRENT_VIDEO_FILE = os.path.join(TESTS_DATA_DIR, "Night.Of.The.Living.Dead_1080p_archive.torrent")
-TORRENT_VIDEO_FILE_INFOHASH = binascii.unhexlify("0d6bbc31db060e7082f85a87194d4273aea9924c")
+TORRENT_VIDEO_FILE_INFOHASH = binascii.unhexlify("90ed3962785c52a774e89706fb4f811a468e6c05")
```

Comment:
That old info hash (`0d6bbc31db060e7082f85a87194d4273aea9924c`) was wrong and didn't correpond to Night of the living dead. I found this out when the test was hanging. After I determined the right info hash and replace it, it started to find the torrent in the list of downloads.